test/playback.test.js

```diff
@@ -0,0 +1,74 @@
+import QUnit from 'qunit';
+import videojs from 'video.js';
+/* eslint-disable no-unused-vars */
+import { Hls } from '../src/videojs-contrib-hls';
+
+let when = function(element, type, cb, condition) {
+  element.on(type, function func() {
```

Comment:
Actually I might just use cue points since I haven't really used this function for anything other than doing something at a specific time.